vendor/github.com/HewlettPackard/oneview-golang/ov/uplink_set.go

```diff
@@ -42,22 +43,22 @@ type VlanAttributes struct {
 }
 
 type UplinkSet struct {
+	Category                       string               `json:"category,omitempty"`                       //"category":"uplink-sets",
```

Comment:
As I can see there is no change only rearrangement

Reply:
Yes this is from golang done by Nabhajit...